sdk/AWSXRayRecorder.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>AWSXRayRecorder</id>
-    <version>2.6.1</version>
+    <version>2.6.2</version>
```

Comment:
We need to also update dependencies for this package [here](https://github.com/aws/aws-xray-sdk-dotnet/blob/master/sdk/AWSXRayRecorder.nuspec#L17-L29). No need to increment for ASPNET package. Following are the increments.

For both NET45 and NETCore,
1. Updating AWSSDK dependency to `2.6.2`
2. For rest packages except ASPNET incrementing version to `2.5.2`

In summary, we maintain same minimum version for each package in this file as per the "current" release commit.